load_testing/configs/1_200_ramp_config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
change filename to "10_to_200_user_config.py" if you agree with removing the ramping.

Reply:
Easier to understand